content/en/contribute/docs/style-guide.md

```diff
@@ -4,6 +4,7 @@ linkTitle: "Style Guide"
 weight: 10
 description: >
   Editorial guidelines for writing documentation
+show_toc: true
```

Comment:
hah! love the simplicity of this and all the extra lines an manual maintenance you got rid of.  pro move!

Reply:
One thing of note is that the layout is slightly different: 
<img width="519" alt="image" src="https://github.com/medic/cht-docs/assets/10299338/de0f413e-c9fe-4eb6-ad7e-4e7c71051d43">

Also there's already a ToC on the right-hand side, and this is the only page with a ToC

Reply:
yeah - I think it's fine. Thanks for calling out the difference!